examples/BertNewsClassification/MLproject

```diff
@@ -5,16 +5,16 @@ conda_env: conda.yaml
 entry_points:
   main:
     parameters:
-      epochs: {type: int, default: 1}
+      max_epochs: {type: int, default: 1}
       num_samples: {type: int, default: 15000}
       save_model: {type: bool, default: True}
       vocab_file: {type: str, default: 'https://s3.amazonaws.com/models.huggingface.co/bert/bert-base-uncased-vocab.txt'}
       model_save_path: {type: str, default: 'models'}
 
     command: |
           python news_classifier.py \
-            --epochs {epochs} \
-            --num-samples {num_samples} \
-            --save-model {save_model} \
-            --vocab-file {vocab_file} \
-            --model-save-path {model_save_path} 
+            --max_epochs {max_epochs} \
```

Comment:
Please confirm these are the actual argument names and work

Reply:
Yes.. the argument names are matching and i was able to run the script with the arguments